lib/util/widget/card/card_items.dart

```diff
@@ -0,0 +1,43 @@
+List<CardItem> items = [
+  CardItem(
+    image: "card1.png",
+    title: 'Art & Humanities',
+    subtitle: 'Draw & Paint Arts',
+    duration:
```

Comment:
use Duration(hours:2, min:2, sec:33)

Reply:
I asked u to convert duration data type from String to Duration and use it.
only convert to string where u have to display